server/src/main/java/org/elasticsearch/common/blobstore/BlobContainer.java

```diff
@@ -107,6 +109,20 @@
      */
     void deleteBlob(String blobName) throws IOException;
 
+    /**
+     * Deletes the blobs with giving names. Unlike {@link #deleteBlob(String)} this method will not throw an exception
+     * when one or multiple of the given blobs don't exist and simply ignore this case.
+     *
+     * @param   blobNames  The names of the blob to delete.
+     * @throws  NoSuchFileException if the blob does not exist
```

Comment:
This @throws contradicts "Unlike {@link #deleteBlob(String)} this method will not throw an exception..."

Reply:
removed it :) My bad for just copy and pasting